lightning/src/ln/channelmanager.rs

```diff
@@ -10706,6 +10706,28 @@ where
 			"Dual-funded channels not supported".to_owned(),
 			 msg.channel_id.clone())), *counterparty_node_id);
 	}
+
+	fn handle_message_received(&self) {
+		for (context, invoice_request) in self
+			.pending_outbound_payments
+			.release_invoice_request_awaiting_invoice()
+		{
+			match self.create_blinded_paths(context) {
+				Ok(reply_paths) => match self.enqueue_invoice_request(invoice_request, reply_paths) {
+					Ok(_) => {}
+					Err(_) => {
+						log_info!(self.logger, "Retry failed for an invoice request.");
+					}
+				},
+				Err(_) => {
+					log_info!(self.logger,
```

Comment:
Do we want this as an info line? I was thinking that this could be a good warning?

Reply:
Though retrying is a low-priority operation, I think a `log_warn` might be better than `log_info` in this case!
Thanks for the pointer!
Updated in [pr3010.24](https://github.com/shaavan/rust-lightning/commits/pr3010.24)